scripts/start_container.sh

```diff
@@ -44,8 +44,7 @@ case ${HTTPS} in
 esac
 
 if [ "${useHTTPS}" = true ]; then
-    kjudge -port 443 -file /data/kjudge.db -https /certs
+    kjudge -port 443 -file /data/kjudge.db -https /certs ${@:1}
```

Comment:
```suggestion
    kjudge -port 443 -file /data/kjudge.db -https /certs "$@"
```

Reply:
Looks like $@ really does not include the 0th argument